bentoctl_lambda/aws_lambda/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG BENTOML_VERSION=1.0.0a1
+ARG PYTHON_VERSION=3.8
+FROM bentoml/bento-server:$BENTOML_VERSION-python$PYTHON_VERSION-debian-runtime
```

Comment:
still thinking about the best way to get the bentoml version and python version from a bento